agent/go.mod

```diff
@@ -46,6 +45,7 @@ require (
 	github.com/containerd/continuity v0.3.0 // indirect
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/didip/tollbooth v4.0.2+incompatible // indirect
```

Comment:
General Question!: Why was `github.com/didip/tollbooth v4.0.2+incompatible` moved here?

Reply:
This was done by `go mod tidy`. The reason is that `agent` module no longer has a direct dependency on `tollbooth` which is a library that provides request rate limiting middleware. `ecs-agent` module is responsible for adding the rate limiting middleware to TMDS, so `agent` module now has an indirect dependency on `tollbooth` while `ecs-agent` has a direct dependency (https://github.com/aws/amazon-ecs-agent/blob/c83389c0bd09aa85c521fa52688ece258f82985e/ecs-agent/go.mod#L8).